python/test/test_variableslice.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+
+from dataset import *
+import numpy as np
+
+
+class TestVariableSlice(unittest.TestCase):
+
+    def setUp(self):
+        d = Dataset()
+        d[Coord.X] = ([Dim.X], np.arange(10))
+        d[Data.Value, "a"] = ([Dim.X], np.arange(1,10,dtype=float)) 
+        d[Data.Value, "b"] = ([Dim.X], np.arange(1,10,dtype=float)) 
+        self._d = d
+        self._dslice = d["a"]
```

Comment:
Storing views is discouraged. Essentially it is like storing an iterator, i.e., there is in general a risk of invalidation. It does not happen in this case but I think we should not rely on "bad practices" in our tests, since it will eventually be used as a guideline.